oe_theme.theme

```diff
@@ -1858,6 +1861,42 @@ function _oe_theme_preprocess_site_specific_footer_links(array &$variables): voi
   }
 }
 
+/**
+ * Get the Accessibility statement link.
+ */
+function _oe_theme_preprocess_accessibility_link(array &$variables) {
+  if (!\Drupal::service('module_handler')->moduleExists('oe_corporate_site_info')) {
+    return;
+  }
+  $cacheability = CacheableMetadata::createFromRenderArray($variables);
+  $configuration = \Drupal::configFactory()->get(SiteInformation::CONFIG_NAME);
+  $cacheability->addCacheableDependency($configuration);
+  /** @var \Drupal\oe_corporate_site_info\SiteInformationInterface $site_information */
+  $site_information = \Drupal::service('oe_corporate_site_info.site_information');
+  if (!$site_information->hasAccessibilityLink()) {
+    return;
+  }
+  $uri = $site_information->getAccessibilityLink();
+  $uri_parts = parse_url($uri);
+  if ($uri_parts['scheme'] === 'entity') {
+    [$entity_type_id, $entity_id] = explode('/', $uri_parts['path'], 2);
+    if ($entity_type_id === 'node' && $entity = \Drupal::entityTypeManager()->getStorage($entity_type_id)->load($entity_id)) {
```

Comment:
Load the entity outside the argument and check if it's an instance of NodeInterface.